client/src/pages/index.js

```diff
@@ -1,10 +1,20 @@
 import React from "react";
 import Layout from "../components/Layout";
+import Logo from '../components/Common/Logo';
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  padding: 8px 0;
```

Comment:
Instead of 8px, can you please update this to use spacing8 from the style-constants? 

You'll have to add an import at the top. 
import {spacing8} from "../resources/style-constants";

Reply:
This was just for demo purposes, but I'll fix that up